src/test/java/seedu/club/logic/commands/member/EditMemberDescriptorTest.java

```diff
@@ -48,8 +51,8 @@ public void equals() {
         editedAmy = new EditMemberDescriptorBuilder(DESC_AMY).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(DESC_AMY.equals(editedAmy));
 
-        // different tags -> returns false
-        editedAmy = new EditMemberDescriptorBuilder(DESC_AMY).withTags(VALID_TAG_HUSBAND).build();
+        // different roles -> returns false
+        editedAmy = new EditMemberDescriptorBuilder(DESC_AMY).withRoles(VALID_MEMBER_ROLE_PRESIDENT).build();
```

Comment:
Minor issue, but IDK if this passes the coding standard. The coding standard says 
> Prefer higher-level breaks to lower-level breaks. 

So maybe break based on the dot(.) operator might be safer? IDK how strict they are on this though. Also, 
> Indentation for wrapped lines should be 8 spaces

```suggestion
        editedAmy = new EditMemberDescriptorBuilder(DESC_AMY)
                .withRoles(VALID_MEMBER_ROLE_PRESIDENT).build();
```